src/renderers/dom/test/ReactTestUtilsEntry.js

```diff
@@ -437,40 +437,39 @@ var ReactTestUtils = {
 /**
  * Exports:
  *
- * - `ReactTestUtils.Simulate.click(Element/ReactDOMComponent)`
- * - `ReactTestUtils.Simulate.mouseMove(Element/ReactDOMComponent)`
- * - `ReactTestUtils.Simulate.change(Element/ReactDOMComponent)`
+ * - `ReactTestUtils.Simulate.click(Element)`
+ * - `ReactTestUtils.Simulate.mouseMove(Element)`
+ * - `ReactTestUtils.Simulate.change(Element)`
  * - ... (All keys from event plugin `eventTypes` objects)
  */
 function makeSimulator(eventType) {
-  return function(domComponentOrNode, eventData) {
-    var node;
+  return function(domNode, eventData) {
     invariant(
-      !React.isValidElement(domComponentOrNode),
-      'TestUtils.Simulate expects a component instance and not a ReactElement.' +
-        'TestUtils.Simulate will not work if you are using shallow rendering.',
```

Comment:
Sorry about the churn but I just realized this part is important. I missed it on the first read. Can you please add it to the message as the last sentence for this particular case?

e.g. "Note that TestUtils.Simulate [...]"